app/src/main/java/org/oppia/app/profile/ProfileChooserFragmentPresenter.kt

```diff
@@ -42,6 +73,13 @@ class ProfileChooserFragmentPresenter @Inject constructor(
     return binding.root
   }
 
+  /** Randomly selects a color for the new profile that is not already in use. */
+  private fun getColor(): String {
+    return COLORS_LIST.map {
+      String.format("#%06X", 0xFFFFFF and ContextCompat.getColor(context, it))
```

Comment:
I'm not keen on the string conversion here--can we just use an integer encoded in RGB everywhere, instead? Then we don't need to convert to/from string and int.

Reply:
Updated to now store colorRgb as an integer.